.circleci/config.yml

```diff
@@ -330,3 +384,61 @@ workflows:
           filters:
             branches:
               only: master
+      - gke-create-cluster:
+          cluster_id: "1"
+          name: "gke-cluster1"
+          requires:
+            - build-nsmd
```

Comment:
Can we leverage GKE build?

Reply:
We can, but it was slower in my experiments and require few more tricks and configurations on GKE, since I've reached few times some quotas.